src/ImageSharp/Formats/WebP/LosslessUtils.cs

```diff
@@ -0,0 +1,656 @@
+                            PredictorAdd0(pixelData, startIdx, numberOfPixels, output);
+                            break;
+                        case 1:
+                            PredictorAdd1(pixelData, startIdx, numberOfPixels, output);
+                            break;
+                        case 2:
+                            PredictorAdd2(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 3:
+                            PredictorAdd3(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 4:
+                            PredictorAdd4(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 5:
+                            PredictorAdd5(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 6:
+                            PredictorAdd6(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 7:
+                            PredictorAdd7(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 8:
+                            PredictorAdd8(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 9:
+                            PredictorAdd9(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 10:
+                            PredictorAdd10(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 11:
+                            PredictorAdd11(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 12:
+                            PredictorAdd12(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                        case 13:
+                            PredictorAdd13(pixelData, startIdx, numberOfPixels, width, output);
+                            break;
+                    }
+
+                    x = xEnd;
+                }
+
+                processedPixels += width;
+                ++y;
+                if ((y & mask) is 0)
+                {
+                    // Use the same mask, since tiles are squares.
+                    predictorModeIdxBase += tilesPerRow;
+                }
+            }
+
+            output.CopyTo(pixelData);
+        }
+
+        private static void PredictorAdd0(Span<uint> input, int startIdx, int numberOfPixels, Span<uint> output)
+        {
+            int endIdx = startIdx + numberOfPixels;
+            for (int x = startIdx; x < endIdx; ++x)
+            {
+                uint pred = Predictor0();
```

Comment:
as Predictor0() returns a constant value, it shouldn't be called once per loop iteration